compiler/rustc_middle/src/query/mod.rs

```diff
@@ -1966,4 +1966,11 @@ rustc_queries! {
         eval_always
         desc { "computing the backend features for CLI flags" }
     }
+
+    /// All items in the crate.
+    query hir_crate_items(_: ()) -> rustc_middle::hir::ModuleItems {
```

Comment:
Could you place it with the other HIR-related queries?  For instance above hir_module_items?